src/Bootstrap/less/theme/page-display-package.less

```diff
@@ -117,6 +117,14 @@
     }
   }
 
+  .btn-gh-repo {
+    background-color: white;
+  }
+
+  .btn-gh-repo:hover {
+    background-color: #d9edf7;
```

Comment:
Where did you get this color from? Can we use an already defined variable for this?

Reply:
(Feel free to ping me if you need more info on this comment)